### PR TITLE
Feat/quiz page

### PR DESCRIPTION
**Pull request by Debarshi95**
PR for entire Quiz App and other related pages

Preview: https://627669215dcde472386bed38--debarshib-flixquiz.netlify.app/

### CODE REVIEW

File: src/components/ErrorBoundary/ErrorBoundary.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      return (
+        <div className="flex h-screen items-center justify-center">
+          <Text className="text-3xl text-slate-300">Oops! Some error occurred</Text>
+        </div>
+      );
+    }
+    return children;
+  }
+}
+
```
Comment (pankaj-prog) [inline src/components/ErrorBoundary/ErrorBoundary.js:36]: New to me, I didn't know about handling errors in component rendering before. I always get something to learn while reviewing your code.

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Text } from 'components';
+import { memo } from 'react';
+
+const Navbar = () => {
+  return (
+    <div className="p-6">
```
Comment (pankaj-prog) [inline src/components/Navbar/Navbar.js:6]: you can use nav here

**Review by pankaj-prog — APPROVED** [review]
You have nicely structured your code, it's readable. Awesome work

**Review by m26unkwn — APPROVED** [review]
1. code quality is good
2. everything is well structure
3.  Like the way you are handling the errors
4. Using Modal component is great

You can go ahead with code

**Pull request merged**